scene/main/node.cpp

```diff
@@ -1250,6 +1254,27 @@ bool Node::is_processing_unhandled_key_input() const {
 	return data.unhandled_key_input;
 }
 
+void Node::set_process_unhandled_picking_input(bool p_enable) {
+	ERR_THREAD_GUARD
+	if (p_enable == data.unhandled_picking_input) {
+		return;
+	}
+	data.unhandled_picking_input = p_enable;
+	if (!is_inside_tree()) {
+		return;
+	}
+
+	if (p_enable) {
+		add_to_group("_vp_unhandled_picking_input" + itos(get_viewport()->get_instance_id()));
+	} else {
+		remove_from_group("_vp_unhandled_picking_input" + itos(get_viewport()->get_instance_id()));
+	}
```

Comment:
It should also be added/removed to/from such group on `NOTIFICATION_ENTER_TREE`/`NOTIFICATION_EXIT_TREE` like it's done for other similar groups (the node can be readded into a different viewport).

Reply:
Thanks!